CMakeLists.txt

```diff
@@ -1360,7 +1364,7 @@ _add_if("brotli"        HAVE_BROTLI)
 _add_if("zstd"          HAVE_ZSTD)
 _add_if("AsynchDNS"     USE_ARES OR USE_THREADS_POSIX OR USE_THREADS_WIN32)
 _add_if("IDN"           HAVE_LIBIDN2)
-_add_if("Largefile"     (CURL_SIZEOF_CURL_OFF_T GREATER 4) AND
+_add_if("Largefile"     (SIZEOF_CURL_OFF_T GREATER 4) AND
```

Comment:
This seems to fix detection for Largefile support in CMake, but for some reason curl is still not build with it, see AppVeyor CI.

Reply:
This is an incorrect fix. Both `curl_off_t` and `off_t` must be larger than 4, **or** `USE_WIN32_LARGE_FILES` should be defined. Why does this commit change anything? It would seem to indicate an earlier problem in variable type size detection?

Reply:
No wait, I'm misreading this. Sorry I'm wrong.

Reply:
I think you should extract this Largefile fix into a separate and independent PR

Reply:
I created #6702 for this.

Reply:
Thanks! So I will remove that part from this PR.